### DISCUSSION TITLE
unable to view image files located in a repositories.

### DISCUSSION THREAD

**Discussion by arasamannar** [community/community#69666] [Repositories] [answered]
### Select Topic Area

Question

### Body

unable to view image files located in a repositories.
is there any option to view images present in the following repositories?
I am new to GitHub, please direct me to correct solution.
repo: https://github.com/rvsp/HTML-CSS/blob/master/Tasks/Github-profile-page-design.jpg
attached the error message.

![Screenshot 2023-10-10 132448](https://github.com/community/community/assets/17942179/d64639a1-00b3-4063-856b-7e5d6da64fe1)

**Comment by vishalkrishna8** [comment]
Hi @arasamannar , 
Sometimes, browser extensions or settings can interfere with the display of images. Try viewing the repository in a different browser or with extensions disabled. If you think everything is nice than you should be able to view them directly in the GitHub repository.
Let me know if this was helpful to you.

**Comment by arasamannar** [comment]
Still receiving the same error Vishal tried Opera, Firefox, Chrome, and Edge browsers. but received the same error in all the browsers.

![Screenshot 2023-10-10 141921](https://github.com/community/community/assets/17942179/4b9f9887-7675-433f-980e-a4367518f3e4)

maybe there is an option which i need to enable in GitHub portal to view the files clearly.

**Comment by vishalkrishna8** [comment]
I repeat all the steps again, please check one by one
* Go to the GitHub repository where the image files are located.
* Find the image file you want to view within the repository.
* Click on the image file's name, and GitHub will display the image in your browser. You can click on the image to zoom in or out.
![gitimg](https://github.com/community/community/assets/121739011/79d7bc90-4ebf-44e2-8098-e46633c2565e)

If you still can't view images, check the permissions and browser settings.

**Comment by arasamannar** [comment, marked as answer]
I have found the reason for the error it is because of my Network provider when I changed my network it worked.
